test/fixtures.py

```diff
@@ -182,6 +182,7 @@ def appconfig(database_uri):
         "MAIL_SERVER": "",
         "MAIL_DEFAULT_SENDER": "admin@example.com",
         "DATABASE_SECRET_KEY": "anothercrazykey!",
+        "FEATURE_PROXY_CACHE": True,
```

Comment:
You'll also need to set it in `config.py`, defaulting to False there.
And define the schema in `util/config/schema.py`

Reply:
That's already done - am I missing anything?

Reply:
Noticed I had missed setting it in the schema - fixed.